gprofiler/profilers/java.py

```diff
@@ -992,6 +1048,8 @@ def _profile_ap_process(self, ap_proc: AsyncProfiledProcess, comm: str, duration
             # Process still running. We will stop the profiler in finally block.
             pass
         else:
+            if self._collect_process_run_times is not None:
+                process_uptime_str = uptime_calculator.get_alive_time_bin(self._collect_process_run_times)
```

Comment:
Don't we want `process_uptime_str` to be set also when the process doesn't exit? Such that we see what happened in its early lifetime